Cesium3DTilesSelection/src/Tileset.cpp

```diff
@@ -1416,6 +1432,16 @@ void Tileset::_unloadCachedTiles() noexcept {
     }
 
     pTile = pNext;
+
+    // A time budget of 0.0 indicates we shouldn't throttle cache unloads.
+    std::chrono::time_point<std::chrono::system_clock> time =
+        std::chrono::system_clock::now();
+    if (timeBudget > 0.0 &&
+        std::chrono::duration_cast<std::chrono::duration<double, std::milli>>(
+            time - start)
+                .count() >= timeBudget) {
```

Comment:
This can be simplified by computing the _end_ time once outside the loop, and then just checking the current time against it inside the loop. The unlimited budget case can be handled by setting the end time to infinity or max value or a million years from now or whatever is convenient.

Reply:
Good idea, it looks cleaner now! Also switched out those long, verbose std::chrono types with "auto", if that's okay.